### PR TITLE
ENH: Updates for lightpath

### PR DESCRIPTION
**Pull request by ZLLentz**
## Description

Accumulated updates to make devices work in the KFE lightpath

## Motivation and Context

They asked for a lightpath, and it will be more useful if all the elements work

## How Has This Been Tested?

Interactively

## Where Has This Been Documented?

Pre-release notes

## Pre-merge checklist
- [x] Code works interactively
- [x] Code contains descriptive docstrings, including context and API
- [x] New/changed functions and methods are covered in the test suite where possible
- [x] Test suite passes locally
- [x] Test suite passes on travis
- [x] Ran docs/pre-release-notes.sh and created a pre-release documentation page
- [x] Pre-release docs include context, functional descriptions, and contributors as appropriate

### CODE REVIEW

**Comment by ZLLentz** [comment]
I'd like to merge this bit (after cleaning up docs) because I haven't made a change to this in a day or so

File: pcdsdevices/slits.py [hunk 1]
```diff
@@ -438,7 +438,7 @@ class BeckhoffSlits(SlitsBase):
 
     def __init__(self, prefix, *, name, **kwargs):
         self._started_move = False
-        super().__init__(prefix, name=name, **kwargs)
+        super().__init__(prefix, name=name, nominal_aperture=3, **kwargs)
```
Comment (klauer) [inline pcdsdevices/slits.py:441]: Related nice explanation of this parameter is in the `LusiSlits`:

```
    ... we set a
    `nominal_aperture` for each unique device along the beamline. This is
    value is considered the smallest the slit aperture can become without
    blocking the beamline. Both the `xwidth` and the `ywidth`(height) need to
    exceed this `nominal_aperture` for the slits to be considered removed.
```

and the check is pretty simple in `SlitsBase`, where widths for the case of `BeckhoffSlits` is `xwidth` and `ywidth`:

```python
        self._inserted = (min(widths) < self.nominal_aperture.get())
```
```

**Review by klauer — APPROVED** [review]
I think this looks good. Comment does not require any action, just a note as I read through what this PR was doing.

File: pcdsdevices/slits.py (hunk 1 above)
Comment (ZLLentz) [reply pcdsdevices/slits.py:441]: Might want to move this parameter to `happi` at some point- not particularly in a hurry to do so.

**Pull request merged**